src/pages/profile/ui/component.tsx

```diff
@@ -23,7 +24,7 @@ export const ProfilePage = () => {
               Go to the account settings
             </Button>
 
-            <Button variant="outlined" onClick={() => sessionModel.logout()}>
+            <Button variant="outlined" onClick={() => sessionModel.logoutFx()}>
```

Comment:
You can't use it like that. you need to 

1. Import the session model into the page page
2. attach the effect extracted from sessionModel 
3. create a logoutRequested event in the page history
4. create the sample that will react to the event and target the attached effect